tgui/packages/tgui/interfaces/PodDebugPanel.tsx

```diff
@@ -0,0 +1,112 @@
+import {
+  Button,
+  Divider,
+  Icon,
+  LabeledList,
+  NoticeBox,
+  ProgressBar,
+  Section,
+} from 'tgui-core/components';
+import { BooleanLike } from 'tgui-core/react';
+
+import { useBackend } from '../backend';
+import { Window } from '../layouts';
+
+type Pod = {
+  name: string;
+  has_cell: BooleanLike;
+  charge: number;
+  maxcharge: number;
+};
+type Part = {
+  ref: string;
+  name: string;
+  slot: string;
+};
+type Data = {
+  pod: Pod;
+  parts: Part[];
+};
+
+export const PodDebugPanel = (props) => {
```

Comment:
use a function please, this is inconsistent

Reply:
what is inconsistent